android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt

```diff
@@ -63,6 +63,7 @@ class VideoPlaybackService : MediaSessionService() {
 
         mediaSessionsList[player] = mediaSession
         addSession(mediaSession)
+        startForeground(mediaSession.player.hashCode(), buildNotification(mediaSession))
```

Comment:
startForground need a specific permission starting from android P:

Apps targeting API Build.VERSION_CODES.P or later must request the permission android.Manifest.permission.FOREGROUND_SERVICE in order to use this API.

I think you need at least to update the documentation to indicate to enable this option

Reply:
Already present, good eye, though! https://thewidlarzgroup.github.io/react-native-video/component/props#shownotificationcontrols